data-esnext.js

```diff
@@ -310,9 +326,10 @@ exports.tests = [
     },
     {
       name: 'SIMD.%type%.check',
+      spec: 'https://tc39.github.io/ecmascript_simd/#simd-check',
       mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SIMD/check',
       exec: function(){/*
-        return typeof SIMD.Float32x4.check === 'function';
+        return typeof SIMD.Float32x4.check === 'function' && typeof SIMD.Int32x4.check === 'function' && typeof SIMD.Uint32x4.check === 'function' && typeof SIMD.Bool32x4.check === 'function';
```

Comment:
Sorry, I am not a ninja JavaScripter like you guys.  I tried doing like this:
```
return 'Float32x4,Int32x4,Uint32x4,Bool32x4'.split(',').every(function(type){
    return typeof SIMD.[type].check === 'function';
});
```
But it always fails.